hack/jenkins/test-flake-chart/report_flakes.sh

```diff
@@ -85,7 +85,11 @@ awk -F, 'NR>1 {
   | sort -g -t, -k2,2 \
   >> "$TMP_FAILED_RATES"
 
-FAILED_RATES_LINES=$(wc -l < "$TMP_FAILED_RATES")
+# Filter out arm64, macOS and windows tests until they're more stable
+TMP_FAILED_RATES_FILTERED=$(mktemp)
+grep -v "arm64\|macOS\|Windows" "$TMP_FAILED_RATES" > "$TMP_FAILED_RATES_FILTERED"
```

Comment:
I think instead of filtering out Windows

lets filterout Crio and Arm64

Reply:
done